### PR TITLE
#161966609 users profile

### PR DESCRIPTION
**Pull request by verenceLola**
### What does this pull request do?
 Add the users profile feature
### Description of the tasks to be completed?
- add functionality to enable users profile to be created upon successful registration.
 - add functionality to enable user to view profiles of other users and their own.
 - add functionality to enable users to edit only their profiles
 - add tests to test the above functionalities.
### How should this be manually tested?
- clone the repo `git clone https://github.com/andela/ah-technocrats.git`
- `cd ah-technocrats`
- checkout out to this branch `git checkout ft-user-profile-16196609 `
- make virtual environment `virtualenv venv`
- activate the virtual environment `source venv/bin/activate`
- install all the dependencies from the requirements file ` pip install -r requirements.txt`
- make migrations  `python manage.py makemigrations`
- apply the migrations `python manage.py migrate`
- run the server  `python manage.py runserver`
- go to Postman to confirm the functionalities.
-  the functionalities can also be tested automatically: run ` ./manage.py test authors/apps/profiles`
### Any background context you want to provide?
- Upon successful login of the user, the user profile should be automatically created. When a user is logged in, he/she should be allow to update the profile as one desires. 
- Authenticated users should be allow to see the profile of other users but should not be able to edit other user's profiles.
### What are the relevant Pivotal Tracker Stories
[#161966609](https://www.pivotaltracker.com/story/show/161966609)
# Checklist:
- [x] My code follows the style guidelines of this project
- [ ] At least 2 people have reviewed my PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My PR has one commit.
### GET USER PROFILE
screenshot 2018-12-04 at 21 00 59

### UPDATE USER PROFILE
screenshot 2018-12-05 at 13 51 50

### CODE REVIEW

**Comment by SilasKenneth** [comment]
Your branch seems to be behind, have you rebased @verenceLola?

**Comment by Mnickii** [comment]
No screenshots

**Comment by salma-nyagaka** [comment]
Why are your tests failing?

**Comment by salma-nyagaka** [comment]
Try to skip the tests that you are not using by adding this decorator **@unittest.skip("Skip this class")** before the classes

**Comment by salma-nyagaka** [comment]
Then do not forget to squash your commits

**Comment by SnyderMbishai** [comment]
Good work!

**Comment by salma-nyagaka** [comment]
Good work, I like the way you have detailed your PR

**Review by salma-nyagaka — COMMENTED** [review]
Good work, I like the way you have detailed your PR

**Comment by verenceLola** [comment]
> Good work, I like the way you have detailed your PR

Thank you, @salma-nyagaka

File: authors/apps/profiles/migrations/0004_auto_20181204_1104.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1 on 2018-12-04 11:04
+
+from django.db import migrations, models
```
Comment (Mnickii) [inline authors/apps/profiles/migrations/0004_auto_20181204_1104.py]: Let's just have one migration file for profile model

Comment (verenceLola) [reply authors/apps/profiles/migrations/0004_auto_20181204_1104.py]: I have made those changes.

**Review by Mnickii — APPROVED** [review]
LGTM

**Pull request merged**